backend/onedrive/quickxorhash/quickxorhash.go

```diff
@@ -199,5 +161,6 @@ func (q *quickXorHash) BlockSize() int {
 func Sum(data []byte) [Size]byte {
 	var d quickXorHash
 	_, _ = d.Write(data)
-	return d.checkSum()
+	sum := d.checkSum()
+	return *(*[Size]byte)(unsafe.Pointer(&sum))
```

Comment:
Can you remove the use of unsafe? I've tried to avoid it in rclone code.

I don't think it will affect performance since it is only called to read the output.